doc/man1/openssl-asn1parse.pod

```diff
@@ -8,21 +8,21 @@ openssl-asn1parse - ASN.1 parsing tool
 
 B<openssl> B<asn1parse>
 [B<-help>]
-[B<-inform PEM|DER>]
-[B<-in filename>]
-[B<-out filename>]
+[B<-inform> I<DER>|I<PEM>]
```

Comment:
Why should DER/PEM be in italics?

Reply:
Because it is a literal argument.  ping @levitte for backup :)

Reply:
Oh wait, damn, those *should* be bold. The reason is that they are names to be replaced some values, but literal values. Damn.

Reply:
That is not the answer I was hoping for, Richard. :)  Ah well.  Fixup commit pushed.  It only fixes PEM DER ENGINE SMIME etc.  If there are others... I don't want to know.